apps/arena/lib/arena/game/player.ex

```diff
@@ -34,7 +34,7 @@ defmodule Arena.Game.Player do
 
   def trigger_natural_healings(players) do
     Enum.reduce(players, %{}, fn {player_id, player}, players_acc ->
-      player = maybe_trigger_natural_heal(player)
+      player = maybe_trigger_natural_heal(player, alive?(player))
```

Comment:
The only reason we need `alive?(player)` is because we don't have a `player.status`

(Nothing to fix here, sort of me ranting about it)

Reply:
Yeah, we have an issue created to fix that.